mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -6841,7 +6841,7 @@ mod trezor_tests {
             "method": "electrum",
             "coin": ticker,
             "servers": tbtc_electrums(),
-            "priv_key_policy": { "type": "Trezor" },
+            "priv_key_policy": "Trezor",
```

Comment:
Shouldn't we tag this PR with `breaking-change: RPC` label since we break the payload type of `electrum` RPC?

Reply:
In https://github.com/KomodoPlatform/komodo-defi-framework/pull/2223
and in [changelog for v2.5.0](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2491), priv_key_policy was [updated to object](https://komodoplatform.com/en/docs/komodo-defi-framework/api/common_structures/wallet/#priv-key-policy). It is good to use consistent types.